tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
     "compilerOptions": {
         "target": "es5",
+        "outDir": "lib/",
         "module": "commonjs",
-        "noEmitHelpers": true,
+        "noEmitHelpers": false,
```

Comment:
It was to prevent emiting the helper twice. Yes, it exists only once now. I'm even considering removing that and extending `Error` class manually.